src/common/util.tsx

```diff
@@ -80,3 +82,21 @@ export const getLocalFocusData = (): string[] => {
   }
   return JSON.parse(localData) as string[];
 };
+
+export const getLocalQuestListDepthData = (): number => {
+  const localQuestListDepth = localStorage.getItem("questDepth");
+  if (localQuestListDepth) {
+    return +localQuestListDepth;
+  }
+
+  return 0;
+}
+
+export const getLocalIsLimitingQuestDepthData = (): boolean => {
+  const localQuestListDepthChecked = localStorage.getItem("isLimitingQuestDepth");
+  if (localQuestListDepthChecked) {
+    return JSON.parse(localQuestListDepthChecked);
+  }
+
+  return false;
```

Comment:
This I believe is functionally equivalent
```suggestion
return localStorage.getItem("isLimitingQuestDepth") === "true";
```